projects/plugins/jetpack/tests/e2e/lib/search-helper.js

```diff
@@ -0,0 +1,397 @@
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+/**
```

Comment:
Tiny nit: add whitespace line before this docblock.

Reply:
fixed in https://github.com/Automattic/jetpack/pull/19447/commits/bc980aa3e00c3a408353131bfd2b8d9497eb2e4e